EchoSystem.axs

```diff
@@ -397,6 +398,7 @@ volatile char ES_eDeviceType
 volatile char ES_eDeviceQuery
 volatile long ES_iDeviceRefreshTime[] = {0}
 volatile long ES_iDeviceHealthTime[] = {60000}  //Every minute
+volatile long ES_iProgramLoopTime[] = {10000} // Every 10 seconds

```

Comment:
This should be 1000 (every second).  The purpose of the original "wait 10" was to trip every second, decrementing the time remaining counter.

Reply:
I was actually trying to get ahold of you, are you on LinkedIn?

Reply:
No, not on LinkedIn.  My email should be visible in my GitHib  profile, I think.  Feel free to try that.  If not, I'll open a discussion board on this project and we can chat there.